Getting-Started/index.md

```diff
@@ -82,7 +82,7 @@
 			</div>
 			<div class="col-sm-6">
 				<h5><a href="Data/Creating-Media/">Creating media</a></h5>
-				<small>Uploading files to media library, using upload fields and image cropper</small>
+				<small>Uploading files to the media library, using upload fields and an image cropper</small>
```

Comment:
As the Image cropper is a feature, I think it makes sense say the following instead. 
```suggestion
				<small>Uploading files to the media library, using upload fields and the image cropper</small>
```
What do you think?